test/test_build_frontend_args.py

```diff
@@ -19,12 +19,11 @@ def test_build_frontend_args(tmp_path, capfd, frontend_name):
     project.generate(project_dir)
 
     # the build will fail because the frontend is called with '-h' - it prints the help message
+    add_env = {"CIBW_BUILD_FRONTEND": f"{frontend_name}; args: -h"}
+    if utils.platform == "pyodide":
+        add_env["TERM"] = "dumb"  # disable color / style
```

Comment:
Hopefully this works? I have no idea why it passes locally though??

Reply:
Seems that it fixed it indeed =)

Reply:
it seems too have been style/color as you pointed out.

Reply:
Maybe something is checking for the GitHub actions var and assuming it is in a CI run that supports color?